src/main/java/ch/uzh/ifi/hase/soprafs22/utilities/InvitationCodeGenerator.java

```diff
@@ -0,0 +1,40 @@
+package ch.uzh.ifi.hase.soprafs22.utilities;
+
+import org.springframework.web.client.RestTemplate;
+
+import java.util.Random;
+
+/**
+ *
+ */
+public class InvitationCodeGenerator {
```

Comment:
Couldn't we make this a static class, as we  save the qrcode byte in the loby and the string also. Thus we could just use two static methods to generate getQR and getAlphanumeric?

Reply:
<img width="834" alt="Bildschirmfoto 2022-04-13 um 15 00 54" src="https://user-images.githubusercontent.com/91155454/163185713-a20a52c8-815e-4fca-a741-f5fc9ee7f9e5.png">

Only nested classes in Java are allowed to be static. To make the class static would contradict my approach to reduce the responsibilities of the Lobby. I'll try to make the methods static in a sec

Reply:
great idea, thx!
looks less redundant now